app/src/main/java/dev/ukanth/iconmgr/util/PackageComparator.java

```diff
@@ -16,6 +16,12 @@
 
 public class PackageComparator implements Comparator<IPObj> {
 
+    private Context ctx;
```

Comment:
You need to be careful when passing around context. Sometimes passing the context using "this" keyword from activity crashes on multiple devices